docs/website.md

```diff
@@ -0,0 +1,14 @@
+# Website Deployment
+
+The [HELM website](https://crfm.stanford.edu/helm/) consists entirely of static content. To build the website, make a copy of the static web assets folder provided by this package, place the `benchmark_output` directory immediately under this server, then serve the folder with a web server. The command `helm-server --print-static-assets-path` will print the path to the static web assets directory. Take care to resolve symbolic links when copying files from this directory.
```

Comment:
Could we provide a bit more concrete context?  First, I'd say that you can just run `helm-server` to serve the website.  Then point out that it's just serving the files (static assets, `benchmark_output` - give the paths for concreteness).  Finally say that you can copy these into an existing server or push it to a GitHub page (perhaps give commands for that).